uberduck_ml_dev/utils/audio.py

```diff
@@ -37,6 +37,17 @@
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 """
 
+# Cell
+from ..models.common import MelSTFT
+
+def sample(mel, algorithm="griffin-lim", **kwargs):
```

Comment:
How about `mel_to_audio` instead of `sample` (outside of the context of a training run `sample` seems less fitting)